api/Startup.cs

```diff
@@ -56,8 +58,9 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 );
             services.AddSingleton<HashPasswordsService>();
+            services.AddSingleton<JwtService>();
 
-            var mapperConfig = new MapperConfiguration(mc =>
+            MapperConfiguration mapperConfig = new (mc =>
```

Comment:
A bit confused about that line. I would say it's pretty hard to read. At least the following code style:
```suggestion
            MapperConfiguration mapperConfig = new(mc =>
```